Applications/Games/Dishonored 2/Steam/script.json

```diff
@@ -0,0 +1,15 @@
+{
+    "scriptName"                 : "Steam",
+    "id"                         : "steam", 
+    "compatibleOperatingSystems" : [
+    	"MACOSX",
+        "LINUX"
+    ],
+    "testingOperatingSystems" : [
+    	"MACOSX",
```

Comment:
I am pretty sure dxvk will not work under macOS, does the game works without it (so that for macOS user will not use dxvk) ?

Reply:
@ImperatorS79 Negative, won't open on DirectX9 -> results in 
![image](https://user-images.githubusercontent.com/11302521/52162211-b7c98080-26d0-11e9-9ba5-a7fa06bbf9e0.png)

DirectX10/11 tested on DXVK work with no issues excluding problem with shaders, untested on PBA

DirectX12 on vkd3d untested

research here: https://github.com/RXT067/Research/tree/master/WINE-Dishonored2

Reply:
But does it works with DirectX11 without DXVK ?